src/harrastuspassi/harrastuspassi/management/commands/import_lipas.py

```diff
@@ -0,0 +1,127 @@
+# -*- coding: utf-8 -*-
+import datetime
+import time
+import logging
+import requests
+from django.contrib.gis.geos import Point
+from django.core.management.base import BaseCommand
+from django.db import transaction
+from harrastuspassi import settings
+from harrastuspassi.models import Hobby, HobbyCategory, HobbyEvent, Location
+
+
+LOG = logging.getLogger(__name__)
+REQUESTS_TIMEOUT = 15
+
+
+class Command(BaseCommand):
+    help = 'Import data from Lipas'
+    source = 'lipas'
+
+    category_mappings = {
+        1180: 'Frisbeegolf',
+        4401: 'Kävely',
+        4403: 'Kävely',
+        4402: 'Hiihto',
+        1520: 'Luistelu',
+        4404: 'Retkeily',
+        4421: 'Moottoriurheilu',
+        4422: 'Moottoriurheilu',
+        4412: 'Pyöräily',
+        4405: 'Retkeily',
+        3230: 'Uinti',
+        1130: 'Kuntosalit',
+        4452: 'Melonta',
+        1210: 'Yleisurheilu',
+    }
+
+    def add_arguments(self, parser):
+        parser.add_argument('--url', action='store', dest='url', default=settings.LIPAS_URL,
+                            help='Import from a given URL')
+
+    def handle(self, *args, **options):
+        page_number = 1
+        created_hobbies = 0
+        updated_hobbies = 0
+        created_locations = 0
+        updated_locations = 0
+
+        self.stdout.write(f'Starting to pull data from {options["url"]}')
+        start_time = time.time()
+        with requests.Session() as session, transaction.atomic():
+            session.headers.update({'Accept': 'application/json'})
+            self.session = session
+
+            while True:
+                response = self.session.get(
+                    options['url']
+                    + f'?fields=type.name&fields=name&fields=freeUse&fields=location.city.name&fields=location.postalCode&fields=location.postalOffice&fields=location.coordinates.wgs84&fields=location.address&fields=type.typeCode&page={page_number}',
```

Comment:
Better not to build query strings by hand.

``` python
query_params = {'foo': 'bar'}
self.session.get(options['url'], params=query_params)
```

https://requests.readthedocs.io/en/master/user/quickstart/#passing-parameters-in-urls